ando/engine.py

```diff
@@ -230,59 +228,45 @@ def check_Path(names, verbose):
 
         if not is_rawdata(names):
             try:
-                raise RawDataError(names)
-            except RawDataError as e:
+                raise DataError(names, type='raw')
+            except DataError as e:
                 if verbose is True:
                     print(e.strerror)
                 out.append(e.strout)
                 bool_error = 1
         if not is_derivatives(names):
             try:
-                raise DerivativeDataError(names)
-            except DerivativeDataError as e:
+                raise DataError(names)
```

Comment:
any reason why here, the DataError is raised without the optional argument "type"? in the two other cases, you have `type=raw` and `type=metadata`, maybe it would be more better to be consistent and have `type=derivative` here?

Reply:
fixed via 815310d